pkgs/tools/graphics/flam3/default.nix

```diff
@@ -19,6 +19,6 @@ stdenv.mkDerivation rec {
     homepage = "https://flam3.com/";
     maintainers = [ maintainers.nand0p ];
     platforms = platforms.linux;
-    license = licenses.cc-by-nc-sa-20;
+    license = licenses.gpl3Only;
```

Comment:
Looks like `gpl3Plus`: https://github.com/scottdraves/flam3/search?q=license&unscoped_q=license

Reply:
Huh, you're quite right - the README doesn't specify 'any later version' but indeed each individual source file does.

Reply:
If you can, please ask them to update the website/readme.

Reply:
Good idea, https://github.com/scottdraves/flam3/pull/32